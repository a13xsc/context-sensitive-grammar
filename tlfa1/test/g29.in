       (               {             Q       ,        V       ,     R      ,        B ,          9      ,   N,       C          ,         O,    *   ,      ^ , G  ,          v          ,  I   ,      L      ,   Z   , E       ,      U         ,         Y       ,  W,X         }                ,            {     *          ,       ^   ,         v   ,         9           }      ,           {           (   C       ,vLRIWVZLE^I)         ,(B    , EIIvNEOGQ) ,    ( R         ,       VGOv*NRBL^**EO     )         ,         (        R ,       IOZX*IBvRQ^ )      , (      R   ,        e        )     }                    ,  R      )        
