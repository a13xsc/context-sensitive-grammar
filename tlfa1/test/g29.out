C
G
B
I
E
L
N
Z
Q
W
V
U
Y
X
O
